### ISSUE TITLE
WCAG Accessibility Issues Flagged

### ISSUE THREAD

**Issue by DChan0823** [react-bootstrap-table/react-bootstrap-table2#1684]
Sortable headers are not structured appropriately.
1. Role for the trigger to sort the column is missing
2. Accessible text provided using aria-label is not meaningful. Screen reader users will not be able to understand the meaning of asc/desc
3. aria-sort is not provided on <th> element.

Please update table to satisfy these WCAG Accessibility Issues

**Comment by jonaspraem** [comment]
Just give an option to set the aria-tags.. there should be a {...props} spread on the input field
